joomla/joomla-cms
# [4.0] Move back-to-top to custom module

Pull Request for Issue #25239 

### Summary of Changes

The "Back to Top" button is now a custom module. This allows users to disable it if need be.
The logic has also changed so that even if you have disabled the footer module but have enabled the `back-top` module, it will still display.

### Testing Instructions

1. Install sample data
2. Ensure the "Back to Top" button is enabled
-----------------------------------
1. Disable the footer module
2. Ensure the "Back to Top" button is enabled
